source/includes/api/_dataset.md

```diff
@@ -588,6 +588,12 @@ curl -X POST https://api.resourcewatch.org/v1/dataset \
 }'
 ```
 
+## Notes
```

Comment:
use a `<aside class="notice"> instead (see https://raw.githubusercontent.com/resource-watch/doc-api/e84fbc6ddd9e01cf5a420bc27aebb4c09e29ca93/source/includes/api/_geostore.md line 24). If you use this syntax, the notice will appear on the sidebar index, which is not desired in this case